package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@mui/types": "^7.2.19",
```

Comment:
please try using `"@mui/material": "latest",` here

Reply:
ok

Reply:
Why didn't these changes reflect in package-lock.json?